.changeset/twenty-hounds-sleep.md

```diff
@@ -0,0 +1,19 @@
+---
+"@comet/cms-site": minor
```

Comment:
Isn't this a breaking change, if it changes the default styling of all image blocks in the frontend?
Could this break the styling in existing projects?

Reply:
No i do not think this is a breaking change. If this case was already handled in the project the prop would be set twice now. (If not, it would fix it)